.devcontainer/dockerBuild.sh

```diff
@@ -0,0 +1,14 @@
+#environment setup
+sudo chmod -R a+rwx /etc/bash.bashrc
+echo "export PATH=\"/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin\"" >> /etc/bash.bashrc
+
+#poetry installation
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+#needed for the current shell session
+export PATH="/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin"
+
+sudo chmod -R a+rwx /workspace
+yarn
+yarn setup
+yarn dev
```

Comment:
Looking at how Posthog and Chatwoot do this, it seems like they don't start `yarn dev` automatically.  They just set up the container and the developer would run `yarn dev` themselves in a terminal within VSCode.  The benefit of this is that the `yarn dev` process can fail and sometimes you need to restart it (e.g. when modifying package.json), so it's nice to have the user be in control of when it's run.

Reply:
Posthog does something cool and writes a welcome message into bashrc:
```sh
echo "printf 'Hello 🦔! To start PostHog run this:\n "./ee/bin/docker-ch-dev-web"\n'" > ~/.zshrc
```